undefined reference to `rubinius::jit::RubiniusJITMemoryManager::DefaultCodeSlabSize'

Latest Rubinius from repository failed to build with LLVM 3.5:

$ CXXFLAGS="-std=c++11" ./configure --prefix=$HOME/rbx-test --cc clang --cxx clang++ --llvm-shared
...

$ rake
...
2: CXX vm/vm.cpp
5: CXX vm/vm_thread_state.cpp
Build time: 77.724523845 seconds
1: LD vm/vm
vm/llvm/artifacts/jit_context.cpp.o: In function `unsigned long const& std::max<unsigned long>(unsigned long const&, unsigned long const&)':
/usr/bin/../lib64/gcc/x86_64-unknown-linux-gnu/4.9.1/../../../../include/c++/4.9.1/bits/stl_algobase.h:222: undefined reference to`rubinius::jit::RubiniusJITMemoryManager::DefaultCodeSlabSize'
clang: error: linker command failed with exit code 1 (use -v to see invocation)
Error: clang++ -o vm/vm vm/artifacts/accessor_primitives.cpp.o vm/artifacts/arguments.cpp.o vm/artifacts/auxiliary_threads.cpp.o vm/artifacts/bytecode_verification.cpp.o vm/artifacts/call_frame.cpp.o vm/artifacts/compiled_file.cpp.o vm/artifacts/config_parser.cpp.o vm/artifacts/console.cpp.o vm/artifacts/dispatch.cpp.o vm/artifacts/environment.cpp.o vm/artifacts/exception.cpp.o vm/artifacts/exception_point.cpp.o vm/art
....

--- a/vm/llvm/jit_memory_manager.cpp
+++ b/vm/llvm/jit_memory_manager.cpp
@@ -558,6 +558,10 @@ const size_t RubiniusJITMemoryManager::DefaultCodeSlabSize = 512 * 1024;
 const size_t RubiniusJITMemoryManager::DefaultSlabSize = 64 * 1024;
 
 const size_t RubiniusJITMemoryManager::DefaultSizeThreshold = 16 * 1024;
+#else
+const size_t RubiniusJITMemoryManager::DefaultCodeSlabSize;
+const size_t RubiniusJITMemoryManager::DefaultSlabSize;
+const size_t RubiniusJITMemoryManager::DefaultSizeThreshold;
 #endif
 
 }

